plugins/angular.js

```diff
@@ -65,3 +86,5 @@ angularPlugin._normalizeData = function (data) {
 };
 
 module.exports = angularPlugin;
+// Install plugin when included
+angularPlugin(window.Raven);
```

Comment:
I'd rather not have to get gulp to handle the instantiation. Is it possible with the current PR design to just call it install the plugin manually before calling `Raven.install()`?